better-code/book.toml

```diff
@@ -1,7 +1,6 @@
 [book]
 authors = ["Sean Parent"]
 language = "en"
-multilingual = false
```

Comment:
If this change was going to fix the filed issue, I'd prefer to have the simple PR that makes this one change, and another to improve the handling of dependency versions or whatever.  Are we even sure they're connected?

Reply:
That was a symptom of poor version management. This PR is the proper fix.